components/blitz/src/ome/formats/importer/transfers/AbstractExecFileTransfer.java

```diff
@@ -48,7 +48,7 @@
     /**
      * "Transfer" files by soft-linking them into place. This method is likely
      * re-usable for other general "linking" strategies by overriding
-     * {@link #createProcessBuilder(File, File)} and the other protected methods here.
+     * {@code createProcessBuilder(File, File)} and the other protected methods here.
```

Comment:
Aha. I think I had Eclipse set wrongly to not think protected methods are reported in Javadoc. I'll revert this accordingly.